### ISSUE TITLE
Contact Users form should alert screen readers when new criteria has been added

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1163]
When new criteria has been added to the Contact Users form, screen readers should be alerted to the updated number of users and the new criteria.

**Comment by sudo-may** [comment]
PR: https://github.com/NationalSecurityAgency/skills-service/pull/1368
